src/condor_tests/test_cgroup_oom.py

```diff
@@ -9,6 +9,7 @@
 import pytest
 import os
 
+import htcondor as htcondor2
```

Comment:
This is usually the other way around; are you sure this is what you want to do?  How does this even work?  Didn't we stop building v1 entirely?

Reply:
Good catch.  We only stopped building v1 on mac/windows